.travis.yml

```diff
@@ -1,16 +1,29 @@
 sudo: false
-git:
-  depth: 2
-branches:
-  except:
-    - /^v\d/
+dist: trusty
 language: node_js
-node_js:
-  - iojs
-  - node
-notifications:
-  email: false
+
+env:
+  matrix:
+    - NODE_VERSION="8" NPM_VERSION="5"
+    - NODE_VERSION="8" NPM_VERSION="6"
+    - NODE_VERSION="9" NPM_VERSION="5"
+    - NODE_VERSION="9" NPM_VERSION="6"
+    - NODE_VERSION="10" NPM_VERSION="5"
+    - NODE_VERSION="10" NPM_VERSION="6"
+
+before_install:
+  - pip install --user codecov
+  # Prepare node and npm
+  - nvm install $NODE_VERSION && nvm use $NODE_VERSION
```

Comment:
Use `node_js` key instead. https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions

Reply:
I fully understand your point, but in this way it wouldn't be possible to specify the NPM version, too.

Reply:
Then just always use the latest version.

https://blog.npmjs.org/post/162451604515/the-npm-clis-long-term-support-lts-policy

> The npm CLI project does not have designated LTS releases. The project only regularly does releases to the most recent major release.

Reply:
ping @hypery2k